tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1192,3 +1192,31 @@ Then('the search bar should be visible in the webUI', async function() {
   const isVisible = await client.page.filesPage().isSearchBarVisible()
   assert.strictEqual(isVisible, true, 'Expected search bar to be visible but is not visible')
 })
+
+Then(
+  'the preview image of file {string} should not be displayed in the file list view on the webUI',
+  async function(fileName) {
+    const isPreviewDisplayed = await client.page.FilesPageElement.filesList().isPreviewImageDisplayed(
+      fileName
+    )
+    assert.strictEqual(
+      isPreviewDisplayed,
+      'disabled',
+      'Expected preview image to be disabled but is visible'
+    )
+  }
+)
+
+Then(
+  'the preview image of file {string} should be displayed in the file list view on the webUI',
+  async function(fileName) {
+    const isPreviewDisplayed = await client.page.FilesPageElement.filesList().isPreviewImageDisplayed(
+      fileName
+    )
+    assert.strictEqual(
+      isPreviewDisplayed,
+      'true',
```

Comment:
You are using 'true' here but in p[previous function you are checking for 'disabled'. i think it would be better to use true/false or enabled/disabled to be more consistent